src/control/settings/Settings.cpp

```diff
@@ -838,19 +838,19 @@ void Settings::save() {
     WRITE_BOOL_PROP(disableScrollbarFadeout);
 
     if (this->scrollbarHideType == SCROLLBAR_HIDE_BOTH) {
-        saveProperty("scrollbarHideType", "both", root);
+        xmlNode = saveProperty("scrollbarHideType", "both", root);
     } else if (this->scrollbarHideType == SCROLLBAR_HIDE_HORIZONTAL) {
-        saveProperty("scrollbarHideType", "horizontal", root);
+        xmlNode = saveProperty("scrollbarHideType", "horizontal", root);
     } else if (this->scrollbarHideType == SCROLLBAR_HIDE_VERTICAL) {
-        saveProperty("scrollbarHideType", "vertical", root);
+        xmlNode = saveProperty("scrollbarHideType", "vertical", root);
     } else {
-        saveProperty("scrollbarHideType", "none", root);
+        xmlNode = saveProperty("scrollbarHideType", "none", root);
     }
 
```

Comment:
Very minor, but can you delete this space to make it clear that the comment is attached to `scrollbarHideType`?